### DISCUSSION TITLE
Please could we have a 0.7.0-alpha-2 release for testing?

### DISCUSSION THREAD

**Discussion by chriskilding** [Relm4/Relm4#537] [General]
Please could you create a 0.7.0-alpha-2 release, so that I can test the new FactoryVecDeque / FactoryHashMap builder APIs and add a persistent state management guide to the book?

**Comment by AaronErhardt** [comment]
Sorry, I wanted to publish this earlier, but now 0.7.0-beta.1 is out :)

**Reply by chriskilding** [reply]
Thanks :)
